<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tab_interfaces</class>
 <widget class="QWidget" name="tab_interfaces">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>277</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Interface:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="cmb_interface">
     <item>
      <property name="text">
       <string>Serial Port</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TCP Server</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TCP Client</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_1">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stack">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="page_serialport" name="p_serialport"/>
     <widget class="page_tcpserver" name="p_tcpserver"/>
     <widget class="page_tcpclient" name="p_tcpclient"/>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_run">
     <property name="text">
      <string>RUN</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>page_serialport</class>
   <extends>QWidget</extends>
   <header>interfaces/page_serialport.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>page_tcpserver</class>
   <extends>QWidget</extends>
   <header>interfaces/page_tcpserver.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>page_tcpclient</class>
   <extends>QWidget</extends>
   <header>interfaces/page_tcpclient.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
